vars/satellite_common.yml

```diff
@@ -0,0 +1,27 @@
+---
+rhsm_username: "user@redhat.com"
+rhsm_password: "updatethisvalue"
+
+# Pass the pool-id which would have all the below repos available.
+#
+# Below for example:
+# Satellite Employee Subscriptions brings in all below repos.
+
+# "rhel-7-server-rpms"
+# "rhel-7-server-satellite-maintenance-6-rpms"
+# "rhel-server-rhscl-7-rpms"
+# "rhel-7-server-satellite-6.x-rpms"
+
+# Employee SKU brings in the below repos.
+# "rhel-7-server-ansible-2-rpms"
+
+# Satellite Employee subscriptions for all other Product/Repos.
+rhsm_pool_id1: 8a85f981519abf020151a22d3c387f2a
+# Employee SKU for 'Red Hat Ansible Engine Product'.
+rhsm_pool_id2: 8a85f98960dbf6510160df23e3367451
```

Comment:
I assume it's fine to put pool IDs out on git not wrapped in a secret?

Reply:
I agree with @ehelms.
Do we plan to have some internal repository with some custom or internal configuration in it? If yes, I think that putting these IDs in should be a good start.

Reply:
@ogajduse those values are not sensitive. 

But answering about internal stuff, we will have another repo internally for sensitive stuff.

Any sensitive stuff would be a one-time configuration at one's own end. 

satellite_common_local.yml file, which we would not add to version control.

I will demo stuff next week regarding the workflow, which would help.

Reply:
`cp  satellite_common.yml satellite_common_local.yml`

@ogajduse , good you asked, will do some changes which would make it very clear. Thanks.

Reply:
Also, @ehelms asked a question to make sure it is not sensitive, it does not necessarily mean it is sensitive.
Why is pool-id not sensitive?
Explanation: you can pass in either `pool` or `pool-id` here, for reasons of speed I have used `pool-id`
pool == "Subscription Name" for example: `Red Hat Satellite Employee Subscription` or `Employee SKU`
pool-id == It's corresponding ID.
So, clearly I feel it is not sensitive information in this case.

Reply:
https://docs.ansible.com/ansible/2.5/modules/redhat_subscription_module.html